karate-core/src/main/java/com/intuit/karate/driver/Driver.java

```diff
@@ -107,6 +110,19 @@
     @AutoDef
     Map<String, Object> cookie(String name);
 
+    @AutoDef
```

Comment:
I left this method as default to avoid duplicating the logic in multiple driver implementations. Let me know if this ok, else we can push to respective driver implementations. - Deepak

Reply:
I like the default method approach